packages/venia-ui/lib/components/CheckoutPage/checkoutPage.gql.js

```diff
@@ -1,14 +1,57 @@
 import gql from 'graphql-tag';
 import { CheckoutPageFragment } from './checkoutPageFragments.gql';
+import { ItemsReviewFragment } from './ItemsReview/itemsReviewFragments.gql';
 
-export const CREATE_CART_MUTATION = gql`
+export const CREATE_CART = gql`
     # This mutation will return a masked cart id. If a bearer token is provided for
     # a logged in user it will return the cart id for that user.
     mutation createCart {
         cartId: createEmptyCart
     }
 `;
 
+export const PLACE_ORDER = gql`
+    mutation placeOrder($cartId: String!) {
+        placeOrder(input: { cart_id: $cartId }) {
+            order {
+                order_number
+            }
+        }
+    }
+`;
+
+// A query to fetch order details _right_ before we submit, so that we can pass
+// data to the order confirmation page.
+export const GET_ORDER_DETAILS = gql`
+    query getOrderDetails($cartId: String!) {
+        cart(cart_id: $cartId) {
+            id
+            email
+            total_quantity
+            shipping_addresses {
+                firstname
+                lastname
+                street
+                city
+                region {
+                    label
+                }
+                postcode
+                country {
+                    label
+                }
+
+                selected_shipping_method {
+                    carrier_title
+                    method_title
+                }
+            }
+            ...ItemsReviewFragment
```

Comment:
I reused a fragment here and could possibly do the same for the other pieces of information that are needed, once those PRs are merged.